lib/dndfonts.sty

```diff
@@ -20,4 +20,4 @@
 \newcommand{\dnd@FooterFont}{\normalfont\scshape}
 \newcommand{\dnd@PageNumberFont}{\normalfont}
 
-\newcommand\dnd@DropCapFont{\ArtNouvcfamily}
\ No newline at end of file
+\newcommand\dnd@DropCapFont{\ArtNouvcfamily}
```

Comment:
The check that the linting had failed on was referring to there being a blank line at the end of the file rather than inserting a `\n`. This is automatically supplied by .editorconfig-compatible editors on file save. Which editor are you using?

Reply:
I use emacs. It looks like emacs was eating my last line (I went to last line, killed any remaining text, and emacs would simply eat the entire line). I got an editorconfig package, and it looks like it is taking care of the whitespace issues. (Testing on two computers: linux and mac)